src/Component/App/App.css

```diff
@@ -1,6 +1,10 @@
 * {
-  /* border: 1px solid green; */
   margin: 0;
+  background-image: url(https://wallpaper-house.com/data/out/7/wallpaper2you_147653.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
```

Comment:
Great job putting it in app.css!!! doing this should make it so its their thru all urls right!